docs/source/tagset/elemento-element-citation.rst

```diff
@@ -432,7 +432,7 @@ Exemplos:
     <ref-list>
         <ref id="B10">
             <label>10</label>
-            <mixed-citation>FUNDAÇÃO TROPICAL DE PESQUISAS E TECNOLOGIA “ANDRÉ TOSELLO”. Base de Dados Tropical. 1985. Disponível em: <http://www.bdt.fat.org.br/acaro/sp/>. Acesso em: 30 maio 2002.</mixed-citation>
+            <mixed-citation>FUNDAÇÃO TROPICAL DE PESQUISAS E TECNOLOGIA &quot;ANDRÉ TOSELLO&quot;. Base de Dados Tropical. 1985. Disponível em: &lt;http://www.bdt.fat.org.br/acaro/sp/&gt;. Acesso em: 30 maio 2002.</mixed-citation>
```

Comment:
Gustavosegue o arquivo original (sem revisão) que é onde apresentava o problema. É na referência de nº 10. O problema é com as aspas no nome da fundação e nas entidades (&lt; &gt;) dentro do elemento <mixed-citation>.

Asael C Silvagandhalf_thewhite@live.com



Date: Mon, 4 Jul 2016 13:39:47 -0700
From: notifications@github.com
To: scielo_publishing_schema@noreply.github.com
CC: gandhalf_thewhite@live.com; mention@noreply.github.com
Subject: Re: [scieloorg/scielo_publishing_schema] [!!!Não incorporar!!!] Segundo pacote de revisões do guia SciELO PS. (#247)

In docs/source/tagset/elemento-element-citation.rst:

> @@ -432,7 +432,7 @@ Exemplos:
>      <ref-list>
>          <ref id="B10">
>              <label>10</label>
> -            <mixed-citation>FUNDAÇÃO TROPICAL DE PESQUISAS E TECNOLOGIA “ANDRÉ TOSELLO”. Base de Dados Tropical. 1985. Disponível em: <http://www.bdt.fat.org.br/acaro/sp/>. Acesso em: 30 maio 2002.</mixed-citation>
> +            <mixed-citation>FUNDAÇÃO TROPICAL DE PESQUISAS E TECNOLOGIA &quot;ANDRÉ TOSELLO&quot;. Base de Dados Tropical. 1985. Disponível em: &lt;http://www.bdt.fat.org.br/acaro/sp/&gt;. Acesso em: 30 maio 2002.</mixed-citation>

Eu não consegui reproduzir o problema. @asael-silva pode me mandar o arquivo via email para eu dar uma olhada.


—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub, or mute the thread.


  
  


 
	
      
		Livre de vírus. www.avast.com.